lib/datapackage.py

```diff
@@ -27,30 +27,30 @@ def create_json():
         if script_exists:
             print("Dataset already available. Check the list or try a different shortname")
 
-    contents['title'] = input("Title/Name: ")
-    contents['description'] = input("Description: ")
-    contents['citation'] = input("Citation: ")
-    contents['homepage'] = input("Site/Homepage of dataset: ")
+    contents['title'] = input("Title/Name: ").strip()
+    contents['description'] = input("Description: ").strip()
+    contents['citation'] = input("Citation: ").strip()
+    contents['homepage'] = input("Site/Homepage of dataset: ").strip()
     tags = input("Tags (separated by ';'): ").split(';')
     contents['keywords'] = [tag.strip() for tag in tags if tag.strip() != ""]
     contents['resources'] = []
```

Comment:
this could be refactored 
```
    while script_exists:
        contents['name'] = input("Shortname (Give a unique identifier for script): ")
        script_exists = contents['name'].lower() in short_names
        if script_exists:
            print("Dataset already available. Check the list or try a different shortname")

    contents['title'] = input("Title/Name: ").strip()
    contents['description'] = input("Description: ").strip()
    contents['citation'] = input("Citation: ").strip()
    contents['homepage'] = input("Site/Homepage of dataset: ").strip()
    tags = input("Tags (separated by ';'): ").split(';')
    contents['keywords'] = [tag.strip() for tag in tags if tag.strip() != ""]
    contents['resources'] = []
```